include/z64actor.h

```diff
@@ -100,34 +100,34 @@ typedef struct {
     /* 0x18 */ Vec3f feetPos[2]; // Update by using `Actor_SetFeetPos` in PostLimbDraw
 } ActorShape; // size = 0x30
 
-#define ACTOR_FLAG_0 (1 << 0)
-#define ACTOR_FLAG_2 (1 << 2)
-#define ACTOR_FLAG_3 (1 << 3)
-#define ACTOR_FLAG_4 (1 << 4)
-#define ACTOR_FLAG_5 (1 << 5)
-#define ACTOR_FLAG_6 (1 << 6)
-#define ACTOR_FLAG_7 (1 << 7)
-#define ACTOR_FLAG_8 (1 << 8)
-#define ACTOR_FLAG_9 (1 << 9)
-#define ACTOR_FLAG_10 (1 << 10)
-#define ACTOR_FLAG_ENKUSA_CUT (1 << 11)
-#define ACTOR_FLAG_12 (1 << 12)
-#define ACTOR_FLAG_13 (1 << 13)
-#define ACTOR_FLAG_14 (1 << 14)
-#define ACTOR_FLAG_15 (1 << 15)
-#define ACTOR_FLAG_16 (1 << 16)
-#define ACTOR_FLAG_17 (1 << 17)
-#define ACTOR_FLAG_18 (1 << 18)
-#define ACTOR_FLAG_19 (1 << 19)
-#define ACTOR_FLAG_20 (1 << 20)
-#define ACTOR_FLAG_21 (1 << 21)
-#define ACTOR_FLAG_22 (1 << 22)
-#define ACTOR_FLAG_23 (1 << 23)
-#define ACTOR_FLAG_24 (1 << 24)
-#define ACTOR_FLAG_25 (1 << 25)
-#define ACTOR_FLAG_26 (1 << 26)
-#define ACTOR_FLAG_27 (1 << 27)
-#define ACTOR_FLAG_28 (1 << 28)
+#define ACTOR_FLAG_TARGETABLE (1 << 0) // navi will fly over the actor and it can be Z targeted
+#define ACTOR_FLAG_UNFRIENDLY (1 << 2) // changes targeting behavior for unfriendly actors (sound, links stance)
+#define ACTOR_FLAG_FRIENDLY (1 << 3) // opposite of the unfriendly flag. flag is not checked against in the original game
+#define ACTOR_FLAG_NO_UPDATE_CULLING (1 << 4) // actor will not stop updating even if outside of the uncull zone
+#define ACTOR_FLAG_NO_DRAW_CULLING (1 << 5) // actor will not stop drawing even if outside of the uncull zone
+#define ACTOR_FLAG_NOT_CULLED (1 << 6) // actor is in the uncull zone. flag still updates regardless of the update/draw cull exclusion flags
+#define ACTOR_FLAG_REACT_TO_LENS (1 << 7) // hidden or revealed by lens of truth (depending on room flags)
+#define ACTOR_FLAG_TALK_REQUESTED (1 << 8) // player has requested to talk to the actor
+#define ACTOR_FLAG_HOOK_CAN_CARRY (1 << 9) // brings the actor back to player if hookshoted
+#define ACTOR_FLAG_HOOK_BRING_PLAYER (1 << 10) // brings player to the actor if hookshoted
```

Comment:
```suggestion
#define ACTOR_FLAG_HOOK_CAN_CARRY (1 << 9) // brings the actor back to player if Hookshoted
#define ACTOR_FLAG_HOOK_BRING_PLAYER (1 << 10) // brings player to the actor if Hookshoted
```
I also don't like abbreviating Hookshot much, but I'm sure most people will disagree.

Reply:
i dont think this needs to be capitalized here, its not a name of a thing at this point